src/main/java/com/exadel/discountwebapp/user/service/UserService.java

```diff
@@ -0,0 +1,30 @@
+package com.exadel.discountwebapp.user.service;
+
+import com.exadel.discountwebapp.user.entity.User;
+import com.exadel.discountwebapp.user.exception.UserNotFoundException;
+import com.exadel.discountwebapp.user.mapper.UserMapper;
+import com.exadel.discountwebapp.user.repository.UserRepository;
+import com.exadel.discountwebapp.user.vo.UserResponseVO;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class UserService {
+
+    private final UserRepository userRepository;
+
+    public UserResponseVO findById(long id) throws UserNotFoundException {
```

Comment:
Declaring throws UserNotFoundException is redundant here

Reply:
deleted